src/vs/workbench/contrib/terminal/browser/remoteTerminalService.ts

```diff
@@ -122,6 +132,13 @@ export class RemoteTerminalService extends Disposable implements IRemoteTerminal
 					this._onPtyHostResponsive.fire();
 				}));
 			}
+			this._register(channel.onPtyHostRequestResolveVariables(async e => {
+				const activeWorkspaceRootUri = historyService.getLastActiveWorkspaceRoot(Schemas.file);
+				const lastActiveWorkspaceRoot = activeWorkspaceRootUri ? withNullAsUndefined(workspaceContextService.getWorkspaceFolder(activeWorkspaceRootUri)) : undefined;
+				const resolveCalls: Promise<string>[] = e.text.map(t => configurationResolverService.resolveAsync(lastActiveWorkspaceRoot, t));
```

Comment:
hard to understand this line. what is `text`?

Reply:
It's the original text, will improve